content/guides/getting-started/component-testing.md

```diff
@@ -31,7 +31,7 @@ and testing a given feature.
 ### Component vs. End-to-End in a Nutshell
 
 We cover the differences between component and end-to-end testing in-depth in
-the ["Choosing your testing type"]() guide.
+the [Choosing your testing type]() guide.
```

Comment:
1. Empty link
2. We don't have a guide by this name

Reply:
Btw there's a [GitHub Markdown Linting Action](https://github.com/marketplace/actions/markdown-linting-action) that catches these kinds of things

Reply:
We do, its in the overview section. I added the link.